Add template for epic (#44)
submariner-io/enhancements: .github/ISSUE_TEMPLATE/epic.md

---
name: Epic

> about: Add an epic issue to the Submariner project

Reviewer: ```suggestion
about: Add an epic issue template to the Submariner project
```

Reviewer 2: Your suggestion doesn't make sense :)

This is the template, the text is what you'll see on GH.

Reviewer: Oh right, please ignore :)

labels: epic

---

> <!-- Please only use this template for submitting epic requests -->

Reviewer: Can you add a description of what is an epic? Something like: "Epics contain features or other work items that help deliver on a specific initiative. Epics are a helpful way to identify and organize the work."

## Epic Description

> Why is this needed:

Reviewer 2: ```suggestion
<!-- Describe why is this work needed and what needs to be done -->
```

## Acceptance Criteria
<!-- Document acceptance criteria here: -->

## Definition of Done (Checklist)
<!-- Make sure to check all relevant items before end of the release: -->

* [ ] Code complete

> * [ ] The acceptance criteria met

Reviewer: What are the acceptance criteria?

Reviewer 2: I think this refers to the block in L13

* [ ] Unit/e2e test added & pass
* [ ] CI jobs pass

> * [ ] Deployed using cloud-prepare+subctl

Reviewer: What about Helm?

* [ ] Deployed using ACM/OCM addon
* [ ] Deployed on supported platforms (for e.g kind, OCP on AWS, OCP on GCP)
* [ ] Run subctl verify, diagnose and gather
* [ ] Uninstall
* [ ] Troubleshooting (gather/diagnose) added
* [ ] Documentation added
* [ ] Release notes added

> 

Reviewer: Metrics?

---
name: Epic
about: Add an epic issue to the Submariner project
labels: epic

---
<!-- Please only use this template for submitting epic requests -->

> ## EPIC Description

Reviewer 3: ```suggestion
## Epic Description
```

Why is this needed:

## Acceptance Criteria
<!-- Document acceptance criteria here: -->

## Work Items
<!-- Add work items here, 

> make sure you have an item to cover all relevant topics from DoD checklist: -->

Reviewer 3: Perhaps we should move this section after.

```suggestion
## Work Items
<!-- Add work items here. 
Make sure you have item(s) to cover all relevant topics from the checklist above: -->
```
